### PR TITLE
split AIP to be different proposal

### PR DESCRIPTION
**Pull request by sherry-x**
### Description

Each AIP should it their own proposal

### CODE REVIEW

**Review by rahxephon89 — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -23,23 +23,23 @@ proposals:
...
+      title: "Enable Block Gas Limit"
+      description: "Enable Block Gas Limit"
     execution_mode: MultiStep
     update_sequence:
-      - FeatureFlag:
-          enabled:
-            - signature_checker_v2_script_fix
-  - name: step_4_enable_block_gas_limit_and_transaction_shuffling
+      - Execution:
+          V3:
```
Comment (bchocho) [inline aptos-move/aptos-release-builder/data/release.yaml:37]:
```
V3:
  block_gas_limit: 35000
  transaction_shuffler_type: no_shuffling
  transaction_deduper_type: txn_hash_and_authenticator_v1
```

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 2]
```diff
@@ -48,8 +48,6 @@ proposals:
           V3:
```
Comment (bchocho) [inline aptos-move/aptos-release-builder/data/release.yaml:48]:
```
V3:
  block_gas_limit: 35000
  transaction_shuffler_type:
    sender_aware_v2: 32
  transaction_deduper_type: txn_hash_and_authenticator_v1
```

**Review by bchocho — COMMENTED** [review]
We'll have to explicitly set the fields in V3. I think it needs to look like this (but please check the yaml syntax @sitalkedia )

File: aptos-move/aptos-release-builder/data/release.yaml (hunk 2 above)
Comment (sherry-x) [reply aptos-move/aptos-release-builder/data/release.yaml:48]: but in this way, if the first step got rejected, this one would enable the other AIP too....

Comment (bchocho) [reply aptos-move/aptos-release-builder/data/release.yaml:48]: Yeah unfortunately the way the code is written I don't think there's a way to avoid this

**Pull request closed without merging**